examples/chart/teleport-cluster/charts/teleport-operator/templates/resources.teleport.dev_roles.yaml

```diff
@@ -880,8 +880,8 @@ spec:
                   description: "Condition contains details for one aspect of the current
                     state of this API Resource. --- This struct is intended for direct
                     use as an array at the field path .status.conditions.  For example,
-                    type FooStatus struct{ // Represents the observations of a foo's
-                    current state. // Known .status.conditions.type are: \"Available\",
+                    \n type FooStatus struct{ // Represents the observations of a
```

Comment:
This `\n` looks odd. Didn't we just have a test failure related to this?

Reply:
Yes, I fixed the test but did not update the manifests we deploy (because we were due to an update anyway with this PR).